tools/buildsys/src/args.rs

```diff
@@ -186,11 +207,13 @@ enum BuildFlags {
     Package = 0b00000001,
     Kit = 0b00000010,
     Variant = 0b00000100,
+    Repack = 0b0001000,
 }
 
 impl From<BuildType> for BuildFlags {
     fn from(value: BuildType) -> Self {
         match value {
+            BuildType::Repack => BuildFlags::Repack,
             BuildType::Package => BuildFlags::Package,
             BuildType::Kit => BuildFlags::Kit,
             BuildType::Variant => BuildFlags::Variant,
```

Comment:
Nit: make the order of the enum variants consistent.

```suggestion
            BuildType::Package => BuildFlags::Package,
            BuildType::Kit => BuildFlags::Kit,
            BuildType::Variant => BuildFlags::Variant,
            BuildType::Repack => BuildFlags::Repack,
```